sql/core/src/test/scala/org/apache/spark/sql/internal/SQLConfEntrySuite.scala

```diff
@@ -164,6 +164,18 @@ class SQLConfEntrySuite extends SparkFunSuite {
     assert(conf.getConf(confEntry) === Some("a"))
   }
 
+  test("checkValue()") {
```

Comment:
thanks.

`FILESOURCE_TABLE_RELATION_CACHE_SIZE` now is a static conf, so we can not change it's value in tests, so we might not be able to test `FILESOURCE_TABLE_RELATION_CACHE_SIZE` specifically? If so, should we remove this test or keep it?